src/OrchardCore.Modules/OrchardCore.Deployment/ViewModels/RecipeFileDeploymentStepViewModel.cs

```diff
@@ -0,0 +1,15 @@
+namespace OrchardCore.Deployment.ViewModels
+{
+    public class RecipeFileDeploymentStepViewModel
```

Comment:
Can we inherit from `RecipeFileDeploymentStep` or use it  because I don't seeven any special member in the view modal?